vmdb/app/models/container_group.rb

```diff
@@ -9,7 +9,7 @@ class ContainerGroup < ActiveRecord::Base
   has_many :containers, :dependent => :destroy
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, :class_name => CustomAttribute, :as => :resource, :conditions => {:section => "labels"}
+  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource
```

Comment:
Fun. so the `extending` syntax is now added inside the block. [docs]

[docs]: http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#module-ActiveRecord::Associations::ClassMethods-label-Association+extensions

sorry, that was a tangent.

I remember extensions being popular around 1.2.x or maybe it was 2.x but people seemed to go towards moving that functionality into scopes on the target